grafana-plugin/src/components/AlertTemplates/AlertTemplatesForm.config.ts

```diff
@@ -1,80 +1,45 @@
+import { AppFeature } from 'state/features';
+
 import { TemplateForEdit, commonTemplateForEdit } from './CommonAlertTemplatesForm.config';
-export interface Template {
-  name: string;
-  group: string;
-}
 
-export const templateForEdit: { [id: string]: TemplateForEdit } = commonTemplateForEdit;
+export const getTemplatesForEdit = (features: Record<string, boolean>) => {
+  if (features[AppFeature.MsTeams]) {
+    return Object.assign({}, commonTemplateForEdit, additionalTemplateForEdit);
```

Comment:
A bit simpler version would be `return { ...commonTemplateForEdit, ...additionalTemplateForEdit };`;